packages/fxa-settings/src/components/InputPassword/index.tsx

```diff
@@ -55,8 +55,8 @@ export const InputPassword = ({
       <button
         type="button"
         data-testid={formatDataTestId('visibility-toggle')}
-        className={`w-5 px-3 py-2 text-grey-600 focus:text-blue-500 box-content ${
-          hasContent ? '-ml-3' : 'hidden'
+        className={`w-5 px-3 py-2 text-grey-600 box-content ${
```

Comment:
It looks like the spacing is off on these buttons (compare the left/right whitespace)

![Screen Shot 2021-04-05 at 10 19 10 AM](https://user-images.githubusercontent.com/6392049/113584301-bbcaf300-95f8-11eb-9fa8-8559d90f06ed.png)

This appears to be due to the `w-5` constraining the width when then size of the icon wants to exceed it:

![Screen Shot 2021-04-05 at 10 22 31 AM](https://user-images.githubusercontent.com/6392049/113584407-dbfab200-95f8-11eb-9d85-b87bb1336cfc.png)

Any reason we can't remove this explicit width and let it auto size? cc @LZoog

Reply:
I agree with your suggestion! thank you for pointing it out. What do you think? @LZoog

Reply:
That sounds good to me! I didn't realize that, so nice call out Jody.

Reply:
Good! I committed the changes